backend/app/src/main/java/com/moh/phlat/backend/esb/json/MaintainFacilityRequest.java

```diff
@@ -0,0 +1,425 @@
+package com.moh.phlat.backend.esb.json;
+
+import ca.bc.gov.health.plr.dto.esb.MaintainProviderRequest;
+import ca.bc.gov.health.plr.dto.facility.esb.CivicAddressDto;
+import ca.bc.gov.health.plr.dto.facility.esb.FacilityDetailsDto;
+import ca.bc.gov.health.plr.dto.facility.esb.FacilityDto;
+import ca.bc.gov.health.plr.dto.facility.esb.FacilityIdentifierDto;
+import ca.bc.gov.health.plr.dto.facility.esb.FacilityNameDto;
+import ca.bc.gov.health.plr.dto.provider.esb.AddressDto;
+import ca.bc.gov.health.plr.dto.provider.esb.NoteDto;
+import ca.bc.gov.health.plr.dto.provider.esb.ProviderDetails;
+import ca.bc.gov.health.plr.dto.provider.esb.TelecommunicationDto;
+
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.moh.phlat.backend.model.Control;
+import com.moh.phlat.backend.model.ProcessData;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.text.DateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.TimeZone;
+import java.util.UUID;
+
+import org.springframework.util.StringUtils;
+
+public class MaintainFacilityRequest implements PlrRequest {
+	
+	public MaintainFacilityRequest(Control control) {
+		
+	}
+	
+	@Override
+	public String processDataToPlrJson(ProcessData data) {
+		try {
+			DateFormat df = JSON_DATE_FORMAT_OJDK11;
+			df.setTimeZone(TimeZone.getTimeZone("UTC"));
+			
+			ObjectMapper om = new ObjectMapper();
+			om.setDateFormat(df);
+			om.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS);
+			om.disable(SerializationFeature.WRITE_DATES_WITH_ZONE_ID);
+			om.disable(DeserializationFeature.ADJUST_DATES_TO_CONTEXT_TIME_ZONE);
+			om.setSerializationInclusion(Include.NON_NULL);
+			
+			return om.writeValueAsString(createMaintainProviderRequest(data));
+		} catch (IOException ex) {
+			System.out.println(ex.getMessage());
+		}
+		return null;
```

Comment:
There wasn't a check for this previously, so it likely would've been caught in the retry loop and by CircuitBreaker. I've updated this so that it instead throws any IOException it gets from attempting to generate the JSON, and then I updated PlrEsbBoundary to wrap the method in a try catch and to automatically skip trying to send the request to PLR; there's no point in putting faulty input through to PLR or through the retry loop, and this should prevent input data issues from unnecessarily tripping the CircuitBreaker